actions/workflows/st2_package_test.yaml

```diff
@@ -0,0 +1,83 @@
+---
+version: '2.0'
+
+st2cd.st2_package_test:
+    type: direct
+    input:
+        - hostname
+        - dns_zone
+        - instance_type
+        - environment
+        - key_name
+        - keyfile
+        - distro
+        - role
+        - release
+        - version
+        - bootstrap_branch
+        - bootstrap_script
+        - debug
+        - st2_username: test
```

Comment:
the username and password is needed here to pass to e2e tests to login an get a token.  we can argue that the bootstrap script should hard code this.  it was "test" at first but now it's "st2admin" @ https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-deb.sh#L5.